my-app/src/Components/Books/Books.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import AddBook from './AllBook';
+import Header from '../Navigation/Header';
+import { getBooks } from '../../redux/books/books';
+import './Books.css';
+
+export default function Books() {
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBooks());
+  }, []);// eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.